samples/artifactory/GENERIC_EVENT/repoStats/worker.ts

```diff
@@ -0,0 +1,65 @@
+import { PlatformContext } from 'jfrog-workers';
+
+interface RepoStatsPayload {
+  paths: string
```

Comment:
why not use an array instead of split ?

Reply:
we are using split instead of an array here as the repoStats plugin is written in that way and since while a user is migrating from plugins to workers it would be a better transition for them if the changes they have to make are bare minimum on their end.